app/layout.tsx

```diff
@@ -2,19 +2,28 @@ import type { Metadata } from "next";
 import "./globals.css";
 import Header from "@/components/Header";
 import Providers from "@/providers/Providers";
+import Head from "next/head";
+import manifest from "@/public/manifest.json";
 
 export const metadata: Metadata = {
   title: "Myco Chat",
-  description: "Generated by create next app",
+  description: manifest.description,
+  themeColor: "#F2E8CC",
```

Comment:
`themeColor` is repeated multiple times in the same file.
- Please DRY your code.
- If possible, replace other occurrences of this variable in other parts of the codebase with your new, `shared` hex variable.